recipe/meta.yaml

```diff
@@ -1,6 +1,5 @@
 {% set version = "4.10.3" %}
```

Comment:
Do you think we should use this PR to update to the latest `conda`? I think it's 4.11 right now.

Reply:
I would prefer to do this in a separate PR and update python to 3.9 at the same time. I have started a separate branch and it works fine.